src/Microsoft.TemplateEngine.Cli/Commands/TemplateCommand.cs

```diff
@@ -190,6 +192,11 @@ private HashSet<string> GetReservedAliases()
             {
                 reservedAliases.Add(alias);
             }
+
+            //TODO: is it possible to get list of aliases for help?
+            reservedAliases.Add("-h");
+            reservedAliases.Add("--help");
```

Comment:
@jonsequitur is it possible to retrieve aliases assigned to help option? I need to know them to reserve them when generating aliases for template options.

Reply:
You can set them yourself so you know what they are using the `CommandLineBuilder.UseHelp` extension method.

It's also indirectly observable programmatically:

![image](https://user-images.githubusercontent.com/547415/142450090-e4adf557-c9a3-46b8-9aec-f30b3b4c41c6.png)

So if you didn't want to diverge from the default, in the short term you could hard-code this list and in order to check for regressions you could write a test that verifies the hard-coded values against the programmatically-produced list.